drivers/ansible_shell/driver.py

```diff
@@ -15,4 +16,9 @@ def initialize(self, context):
     def execute_playbook(self, context, ansible_configuration_json, cancellation_context):
         return self.ansible_shell.execute_playbook(context, ansible_configuration_json, cancellation_context)
 
+    def execute_playbooks(self, context, ansible_configurations_json, cancellation_context):
+        configurations = json.loads(ansible_configurations_json)
+        for configuration in configurations:
+            ansible_configuration_json = json.dumps(configuration)
+            self.ansible_shell.execute_playbook(context, ansible_configuration_json, cancellation_context)
```

Comment:
Please change to:
`
for configuration in configurations: ...
    if (!self.ansible_shell.execute_playbook(context, ansible_configuration_json, cancellation_context)):
        return False;
return True;
`

Reply:
its not needed since AnsibleShell.execute_playbook() itself doesn't return any value... if something goes wrong an exception is thrown so a return value is not expected at the caller side